core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -32,8 +37,6 @@ endif
 
 $(call force,CFG_GENERIC_BOOT,y)
 $(call force,CFG_GIC,y)
-$(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
```

Comment:
`cortex-a15` is used because gcc didn't like `cortex-a53` when the initial port was made, perhaps that has changed now. Juno is actually both A53 and A57.